.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     - name: Analysing the code with pylint
       run: |
-        python -m pylint -E partitionmanager
+        python -m pylint --errors-only partitionmanager
```

Comment:
On the command line `-E` is faster and easier but in scripts and code that others will read, the long form is more self-documenting.